ci-operator/step-registry/sippy-e2e/sippy-e2e-commands.sh

```diff
@@ -0,0 +1,316 @@
+cat << END | sed s/CONTENT/"$(cat /var/run/sippy-ci-gcs-sa/gcs-sa |base64 -w0)"/| oc apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gcs-cred
+  namespace: postgres
+data:
+  openshift-ci-data-analysis-ro: CONTENT
+END
+
+# Get the registry credentials for all build farm clusters out to the cluster-pool cluster.
+# These credentials are in vault and maintained by the TRT team (e.g. for updates and rotations).
+# See https://vault.ci.openshift.org/ui/vault/secrets/kv/show/selfservice/technical-release-team/sippy-ci-dockerconfigjson
+oc -n postgres create secret generic regcred --from-file=.dockerconfigjson=${DOCKERCONFIGJSON} --type=kubernetes.io/dockerconfigjson
+
+# Make the "sippy loader" pod.
+cat << END | oc apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sippy-load
+  namespace: postgres
+  labels:
+    app: sippy
+spec:
+  containers:
+  - name: sippy
+    image: ${SIPPY_IMAGE}
+    imagePullPolicy: Always
+    ports:
+    - name: www
+      containerPort: 8080
+      protocol: TCP
+    - name: metrics
+      containerPort: 2112
+      protocol: TCP
+    resources:
+      limits:
+        memory: 2G
+    terminationMessagePath: /dev/termination-log
+    terminationMessagePolicy: File
+    command:  ["/bin/sh", "-c"]
+    args:
+      - /bin/sippy --load-database --log-level=debug --load-prow=true --load-testgrid=false --release 4.7 --database-dsn=postgresql://postgres:password@postgres.postgres.svc.cluster.local:5432/postgres --mode=ocp --config ./config/openshift.yaml --google-service-account-credential-file /tmp/secrets/openshift-ci-data-analysis-ro ; echo IM_DONE ; sleep 999999
+    env:
+    - name: GCS_SA_JSON_PATH
+      value: /tmp/secrets/openshift-ci-data-analysis-ro
+    volumeMounts:
+    - mountPath: /tmp/secrets
+      name: gcs-cred
+      readOnly: true
+  imagePullSecrets:
+  - name: regcred
+  volumes:
+    - name: gcs-cred
+      secret:
+        secretName: gcs-cred
+  dnsPolicy: ClusterFirst
+  restartPolicy: Always
+  schedulerName: default-scheduler
+  securityContext: {}
+  terminationGracePeriodSeconds: 30
+END
+
+date
+oc -n postgres get po sippy-load
+oc -n postgres describe po sippy-load
+
+is_ready=0
+for i in `seq 1 20`; do
+  echo "${i} Sleeping 30s ..."
+  sleep 30
+  c=$(oc -n postgres logs sippy-load|tail -1|grep IM_DONE|wc -l)
+  if [ $c -eq 1 ]; then
+    echo "sippy load is done."
+    is_ready=1
+    break
+  fi
+  echo "sippy loader pod not done yet ..."
+done
+
+if [ $is_ready -eq 0 ]; then
+  echo "sippy loading never finished on time."
+  exit 1
+fi
+
+# Check to see if we got an error on sippy loader.
+c=$(oc -n postgres logs sippy-load|tail -2|grep "error running command"|wc -l)
+if [ $c -gt 0 ]; then
+  echo "sippy loading got an error."
+  oc -n postgres logs sippy-load
+  exit 1
+fi
+
+date
+oc -n postgres logs sippy-load | tail -10
+
+# Launch the sippy api server pod.
+cat << END | oc apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
```

Comment:
Would love to use our deploymentconfig in continuous-release-jobs here but we can do that one later too.